src/Rbac/Rules/RuleRegistry.php

```diff
@@ -0,0 +1,58 @@
+<?php
+declare(strict_types=1);
+
+/*
+ * Copyright 2010 - 2021, Cake Development Corporation (https://www.cakedc.com)
+ *
+ * Licensed under The MIT License
+ * Redistributions of files must retain the above copyright notice.
+ *
+ * @copyright Copyright 2010 - 2021, Cake Development Corporation (https://www.cakedc.com)
+ * @license MIT License (http://www.opensource.org/licenses/mit-license.php)
+ */
+
+namespace CakeDC\Auth\Rbac\Rules;
+
+/**
+ * Static rule registry to allow reusing rule instances in Rbac permissions
+ */
+class RuleRegistry
+{
+    /**
+     * Rule instances array
+     *
+     * @var array
+     */
+    protected static $rules = [];
```

Comment:
Registry class must have a method to clear $rules, this may affect application unit tests

Reply:
fixed, thanks

Reply:
I don't see changes here

Reply:
sorry, pushed now